### PR TITLE
Final refactoring touch

### PR DESCRIPTION
**Pull request by crazybarber**
* Adding more strict security to logging functions
* Extending logging interface from interface/structure to functions
* Converting ambiguous `schema`  into `project` package
* Throwing out unused functions, consts

### CODE REVIEW

File: internal/cmd/get/get.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func NewCommand(ctx context.CLIContext) *cobra.Command {
 	return cmd
 }
 
-func run(ctx context.CLIContext, opts *options, _ *cobra.Command, args []string) {
+func get(ctx context.CLIContext, opts *options, args []string) {
```
Comment (magickris93) [inline internal/cmd/get/get.go:46]: Maybe `getCommand`? `get` seems to me to be vague.

File: internal/log/level.go [hunk 2]
```diff
@@ -45,8 +45,8 @@ const MaxLevel = SpamLevel
 // LevelNames contains all supported names of logging levels.
 var LevelNames = []string{"disable", "fatal", "error", "warn", "info", "verbose", "debug", "spam"}
 
-// LevelsByName maps level names to golog.Level type.
-var LevelsByName = map[string]Level{
+// levelsByName maps level names to golog.level type.
```
Comment (magickris93) [inline internal/log/level.go:48]: `golog.Level` should remain I think

File: internal/log/log.go [hunk 3]
```diff
@@ -15,170 +14,113 @@ var (
 
 // SetLevel sets minimum level for logs, logs with level above specified value will not be printed.
 func SetLevel(levelName string) {
-	level, ok := LevelsByName[levelName]
+	level, ok := levelsByName[levelName]
```
Comment (magickris93) [inline internal/log/log.go:17]: Is this function correct? What happens if there is no `levelName` in the map? Shouldn't L23 set Error level to `DeafultLevel`? (`level` will be an empty string if `ok` is false)

File: internal/project/util.go [hunk 4]
```diff
@@ -8,25 +8,25 @@ import (
...
 	return config.SaveConfigFile(projectConfig, projectConfig.Meta.Path)
 }
 
-// CreateDefaultProjectConfig creates default ProjectConfig and save it to give path if it's not already there.
-func CreateDefaultProjectConfig(filePath string) (*ProjectConfig, error) {
-	// create default ProjectConfig
-	projectConfig := NewDefaultProjectConfig()
+// CreateDefaultProjectConfig creates default Config and save it to give path if it's not already there.
+func CreateDefaultProjectConfig(filePath string) (*Config, error) {
+	// create default Config
```
Comment (magickris93) [inline internal/project/util.go:26]: Comments in this function seem to be redundant

**Comment by g2a-official** [comment]
**Coverage report**
github.com/g2a-com/klio/internal/dependency/manager/error.go:9:		Error				100.0%
github.com/g2a-com/klio/internal/dependency/manager/manager.go:45:	NewManager			0.0%
github.com/g2a-com/klio/internal/dependency/manager/manager.go:58:	GetUpdateFor			100.0%
github.com/g2a-com/klio/internal/dependency/manager/manager.go:94:	InstallDependency		72.5%
github.com/g2a-com/klio/internal/dependency/manager/manager.go:197:	GetInstalledCommands		100.0%
github.com/g2a-com/klio/internal/dependency/manager/manager.go:222:	downloadFile			68.8%
github.com/g2a-com/klio/internal/dependency/registry/common.go:35:	findHighestMatching		100.0%
github.com/g2a-com/klio/internal/dependency/registry/common.go:53:	isCompatible			100.0%
github.com/g2a-com/klio/internal/dependency/registry/common.go:57:	isMoreSpecific			0.0%
github.com/g2a-com/klio/internal/dependency/registry/local.go:20:	NewLocal			0.0%
github.com/g2a-com/klio/internal/dependency/registry/local.go:29:	Update				92.9%
github.com/g2a-com/klio/internal/dependency/registry/local.go:53:	GetExactMatch			0.0%
github.com/g2a-com/klio/internal/dependency/registry/local.go:57:	GetHighestBreaking		100.0%
github.com/g2a-com/klio/internal/dependency/registry/local.go:61:	GetHighestNonBreaking		100.0%
github.com/g2a-com/klio/internal/dependency/registry/remote.go:22:	NewRemote			0.0%
github.com/g2a-com/klio/internal/dependency/registry/remote.go:31:	Update				90.0%
github.com/g2a-com/klio/internal/dependency/registry/remote.go:67:	GetExactMatch			0.0%
github.com/g2a-com/klio/internal/dependency/registry/remote.go:71:	GetHighestBreaking		100.0%
github.com/g2a-com/klio/internal/dependency/registry/remote.go:75:	GetHighestNonBreaking		100.0%
github.com/g2a-com/klio/internal/dependency/registry/version.go:11:	Match				100.0%
github.com/g2a-com/klio/internal/dependency/registry/version.go:17:	GreaterThan			100.0%
github.com/g2a-com/klio/internal/dependency/registry/version.go:23:	getExactMatch			0.0%
github.com/g2a-com/klio/internal/dependency/registry/version.go:31:	getMinorAndPatchConstraints	100.0%
github.com/g2a-com/klio/internal/dependency/registry/version.go:39:	getMajorConstraints		100.0%
github.com/g2a-com/klio/internal/schema/schema.go:42:			NewDefaultProjectConfig		100.0%
github.com/g2a-com/klio/internal/schema/schema.go:50:			MarshalYAML			60.0%
github.com/g2a-com/klio/internal/schema/schema.go:109:			minimalKlioFile			100.0%
github.com/g2a-com/klio/internal/schema/schema.go:127:			UnmarshalYAML			54.5%
github.com/g2a-com/klio/internal/schema/util.go:11:			LoadProjectConfig		0.0%
github.com/g2a-com/klio/internal/schema/util.go:20:			SaveProjectConfig		0.0%
github.com/g2a-com/klio/internal/schema/util.go:25:			CreateDefaultProjectConfig	83.3%
total:									(statements)			74.4%

**Pull request merged**